apps/omg_child_chain/config/config.exs

```diff
@@ -5,12 +5,12 @@ use Mix.Config
 config :omg_child_chain,
   submission_finality_margin: 20,
   block_queue_eth_height_check_interval_ms: 6_000,
-  fee_file_check_interval_ms: 10_000,
-  fee_buffer_duration_ms: 30_000,
   block_submit_every_nth: 1,
   metrics_collection_interval: 60_000,
+  fee_adapter_check_interval_ms: 10_000,
+  fee_buffer_duration_ms: 30_000,
   fee_adapter: OMG.ChildChain.Fees.FileAdapter,
-  fee_specs_file_name: "fee_specs.json"
+  fee_adapter_opts: [specs_file_name: "fee_specs.json"]
```

Comment:
So here, adding another adapter I generalized the adapter options a little more.
Old file based adapter always read fees from a flat file, this is just making adapter's configuration more generic.

Am I answered you question?

Reply:
so fee_adapter_opts is either `feed_url ` or `specs_file_name `?
and I guess
```
fee_adapter: OMG.ChildChain.Fees.FileAdapter,
fee_adapter_opts: [specs_file_name: "fee_specs.json"]
```
these two things play together, but

```
fee_adapter: OMG.ChildChain.Fees.FileAdapter,
fee_adapter_opts: [feed_url: "fee_specs.json"]
```
this two wouldn't?

Reply:
Correct, each adapter accepts the options it can understand.
Maybe we should bundle these two together to avoid stupid mistakes like:

```
fee_adapter: {OMG.ChildChain.Fees.FileAdapter,
                opts: [feed_url: "<feed service url>"]
}
```

Reply:
do you validate this assumption in the config provider?

Reply:
I guess you can't because the adapter name can be anything

Reply:
Exactly ☝️. Best I can do is to crash FeeServer on init... but bundling them together appears to me more.